lib/cs_guide_web/templates/static_page/edit.html.eex

```diff
@@ -0,0 +1,8 @@
+<section class="center w-80 w-75-l mv6-ns mv5">
+  <h2 class="f4 lh4">Edit Static Page</h2>
+  <%= custom_render_autoform(@conn, :update, [CsGuide.StaticPage], assigns: [changeset: @changeset], update_field: :entry_id, assoc_query: &query/1, exclude:
```

Comment:
I think the update field will be `:page_title` or something similar. See https://github.com/club-soda/club-soda-guide/pull/243/files#r241828029